src/csv/compile-csv-feed.test.ts

```diff
@@ -0,0 +1,34 @@
+import { compileCsvFeedEntry } from './compile-csv-feed';
+import * as datasetFromApi from '../test-helpers/mock-dataset.json';
```

Comment:
Since we are no longer targeting the V3 API with this plug-in, we should probably remove the `dataset` verbage. Maybe something like `geojsonFeatures`?

Reply:
I agree. Renamed to `geojsonFeatures`